signal.c

```diff
@@ -44,6 +44,7 @@
 #include <sys/socket.h>
 #endif
 #include <signal.h>
+#include <stdatomic.h>
```

Comment:
This breaks windows build:

```
D:\a\libevent\libevent\signal.c(47): fatal error C1083: Cannot open include file: 'stdatomic.h': No such file or directory [D:\a\libevent\libevent\build\event_core_shared.vcxproj]
```

https://github.com/libevent/libevent/pull/1105/checks?check_run_id=1245032772#step:5:126

Reply:
@htuch can you address this?

Reply:
I don't currently have the ability to build on Windows, so not sure if I can help with this.